mvpoly/tests/prime.rs

```diff
@@ -353,3 +357,139 @@ fn test_from_variable_column() {
     assert_eq!(p[3], Fp::zero());
     assert_eq!(p[4], Fp::one());
 }
+
+#[test]
+/// As a reminder, here the equations to compute the addition of two different
+/// points `P1 = (X1, Y1)` and `P2 = (X2, Y2)`. Let define `P3 = (X3, Y3) = P1 +
```

Comment:
NitPick: suggest: "here _are_ the equations"

Reply:
Done.